sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RuleDescriptionSection.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.api.server.rule;
+
+/**
+ * Represents a sub-section of a rule description (What's the risk, Assess the risk, etc.)
+ *
+ * @since 9.5
+ */
+public interface RuleDescriptionSection {
+
+  String getKey();
+
+  String getHtmlDescription();
```

Comment:
Would it be better to name it "htmlContent" ?

Reply:
what does you think about this renaming to "htmlContent" @christophe-zurn-sonarsource @damien-urruty-sonarsource @arseniy-sonar  ? Will it be more clear to have "content" inside 'description' instead of a 'description' within a description ?

Reply:
We had the feedback from @christophe-zurn-sonarsource that `htmlContent` was indeed better, so we updated the PR accordingly.